gdpr/gdpr.go

```diff
@@ -24,10 +24,21 @@ type Permissions interface {
 	AuctionActivitiesAllowed(ctx context.Context, bidderCoreName openrtb_ext.BidderName, bidder openrtb_ext.BidderName, PublisherID string, gdprSignal Signal, consent string, aliasGVLIDs map[string]uint16) (permissions AuctionPermissions, err error)
 }
 
-type PermissionsBuilder func(config.GDPR, TCF2ConfigReader, map[openrtb_ext.BidderName]uint16, VendorListFetcher) Permissions
+type GVLVendorIDs map[openrtb_ext.BidderName]uint16
```

Comment:
Fair point. We can get rid of this for now and see if it makes sense to add it back later. Sometimes I prefer a custom type for complex maps even if the type isn't used outside of the file but this isn't complex. It can also be especially beneficial for clarity if there are several related maps defined (see custom map types in `stored_responses.go`). Removed.